native/packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavItem.js

```diff
@@ -81,13 +87,17 @@ const BpkHorizontalNavItem = props => {
     ? BpkTouchableNativeFeedback
     : BpkTouchableOverlay;
   const formattedTitle = isAndroid ? title.toUpperCase() : title;
+  const platformProps = isAndroid
+    ? { background: TouchableNativeFeedback.SelectableBackground() }
```

Comment:
Correct me if I am wrong, but I was under the impression that our usage of `SelectableBackground`/`SelectableBackgroundBorderless` was as follows:
- `(Platform.Version < 21) ? SelectableBackground : SelectableBackgroundBorderless`
- Use `overflow: hidden` if ripple needs to be contained

Reply:
Or just use `SelectableBackground` without clipping. I'm maintaining previous behaviour here. We should reevulate this later